From 6379412f8f972b47c54a5f47e8ff045ab86e5404 Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Thu, 9 Aug 2018 18:30:01 +0300
Subject: [PATCH 0180/1239] arm64: mvebu: config: add pci=pcie_bus_safe to
 default kernel bootargs

setting 'pci=pcie_bus_safe' will set every PCIe device's MPS to the
largest value supported by all devices below the root complex,
This settings leads to optimized PCIe MPS size, and increases
performance for supported PCIe cards.

added extra_params=pci=pcie_bus_safe to the kernel boot string.

Change-Id: Ibd5e0a186f5a0e6f1b1e01c4c4a01be492c76899
Signed-off-by: Igal Liberman <igall@marvell.com>
---
 include/configs/mvebu_armada-common.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index 6018a55664..97ee5fdd04 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -32,6 +32,7 @@
 						"booti $kernel_addr_r " \
 						"$ramdisk_addr_r " \
 						"$fdt_addr_r\0" \
+					"extra_params=pci=pcie_bus_safe\0" \
 					"kernel_addr_r=0x7000000\0"	\
 					"initrd_addr=0xa00000\0"	\
 					"initrd_size=0x2000000\0"	\
-- 
2.29.0

